src/common/Common.h

```diff
@@ -668,13 +669,13 @@ namespace Sapphire::Common
   {
     Common::ActionEffectType effectType;
     Common::ActionHitSeverityType hitSeverity;
-    uint8_t param;
+    uint8_t unk;
     /*!
      * @brief Shows an additional percentage in the battle log
      *
      * Has no effect on what is shown and stored in value
      */
-    int8_t bonusPercent;
+    uint8_t param; // this one is the real param
```

Comment:
more of a note but depending on type, the next 2 (possibly 3 but never could confirm) bytes are basically parameters for the type handler

Reply:
I'm not exactly sure about this but this is where I found the "30" in the packet for sprint.

Reply:
it's the effect param, in the case of sprint it's how fast you run